Integrations/integration-Office_EWS.yml

```diff
@@ -843,6 +843,14 @@ script:
                 lastTime: new Date(new Date().getTime() - (10 * 60 * 1000)).getTime()
             }
         }
+        if (lastRun.folderId !== params.folder) {
+          //Folder was changed, we need to reset lastRun
+          lastRun = {
+              folderId: params.folder,
```

Comment:
Why do you need to put the folder ID on the `lastRun` and not just taking it from the params?

Reply:
Since, if it was changed i want to clear the lastRun and start over again

Reply:
I understand why you restart the `lastTIme` field but why putting the `params.folder` in the last run object?

Reply:
Since then i will not know to compare it with the previous run.
I want to know if the previous run was on the same folder or not